extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -68,4 +72,9 @@ export class AmpStoryRequestService {
           return response.json();
         });
   }
+
+  getBookendElement_(element) {
+    return [].slice.call(element.children)
```

Comment:
Nit: you can use the ``childElementsByTag`` from ``src/dom``

````
return childElementsByTag(this.storyElement_, 'amp-story-bookend')[0];
````

You shouldn't need to pass the element as an argument here, since you're using that private property.
And jsdoc please : ))

Reply:
Even better, `childElementByTag`, since we're only getting one.

Reply:
Nice, thanks! I'll just make the call inline since I don't think it deserves a separate method anymore.